packages/devextreme/testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -4916,39 +4954,124 @@ QUnit.module('reset', () => {
+    [
+        'dxSelectBox',
+        'dxDropDownBox'
+    ].forEach((editorType) => {
+        QUnit.test(`Focused ${editorType} should not lose its value when the form is resized (T1196835)`, function(assert) {
+            let screen = 'lg';
+
+            const name = 'VINET';
+            const value = 'Vins et alcools Chevalier (France)';
+            const form = $('#form').dxForm({
+                formData: { name: name },
+                screenByWidth: function() { return screen; },
+                colCountByScreen: {
+                    sm: 1,
+                    lg: 2
+                },
+                items: [
+                    {
+                        itemType: 'simple',
+                        dataField: 'name',
+                        editorOptions: {
+                            displayExpr: 'Text',
+                            valueExpr: 'Value',
+                            showClearButton: true,
+                            dataSource: [{ Value: name, Text: value }],
+                        },
+                        editorType,
                     },
-                    editorType: 'dxDropDownBox',
+                ]
+            }).dxForm('instance');
+
+            const dropDownEditor = form.getEditor('name');
+
+            screen = 'sm';
+            dropDownEditor.focus();
+            resizeCallbacks.fire();
+
+            assert.strictEqual($(form.getEditor('name').field()).val(), value, `${editorType} contains expected value`);
+        });
+
+        QUnit.test(`Focused ${editorType} inside a tabbed item should not lose its value when the form is resized (T1196835)`, function(assert) {
+            let screen = 'lg';
+
+            const name = 'VINET';
+            const value = 'Vins et alcools Chevalier (France)';
+            const form = $('#form').dxForm({
+                formData: { name: name },
+                screenByWidth: function() { return screen; },
+                colCountByScreen: {
+                    sm: 1,
+                    lg: 2
                 },
-            ]
+                items: [{
+                    itemType: 'tabbed',
+                    tabs: [{
+                        title: 'Phone',
+                        colCount: 1,
+                        items: [{
+                            itemType: 'simple',
+                            dataField: 'name',
+                            editorOptions: {
+                                displayExpr: 'Text',
+                                valueExpr: 'Value',
+                                showClearButton: true,
+                                dataSource: [{ Value: name, Text: value }],
+                            },
+                            editorType,
+                        }],
+                    }],
+                }],
+            }).dxForm('instance');
+
+            const dropDownEditor = form.getEditor('name');
+
+            screen = 'sm';
+            dropDownEditor.focus();
```

Comment:
Is it expected that ddEditor is focused in this test? I thought that we want to focus tabPanel here to text scenario different from the previous test